### ISSUE TITLE
Email Assignee newly assigned Task

### ISSUE THREAD

**Issue by RobH779** [Charlie9830/pounder#18]
Send an email/push notification (toggleable) to assignees they are assigned a task.

**Comment by Charlie9830** [comment]
There are a few reasons that I am hesitant to implement notifications.
### Apple iOS Support
Currently Apple do not support Push notifications on iOS Progressive Web Apps. Hopefully this will change in the future, but a notification system isn't very effective if it is only going to notify Android users.
More info on Apple lagging behind with PWA's here:
https://medium.com/@firt/progressive-web-apps-on-ios-are-here-d00430dee3a7

### My first Cent on Notifications in General
If I were to implement notifications, I would want the receiving user to be able to switch them off or mute them. Especially out of work hours. This would mean that the Source user sending the notification couldn't know for sure if the target user has received it, which in a sense, breaks the concept of Notifications. I could implement a way for the source user to know if the target user has muted notifications, but I feel like this is a bit of privacy issue.

### My second cent.
At the moment I feel like Notifications are out of the scope of Handball. I feel like if a Task is important enough that when assigned to someone they need to be notified immediately, the target should be notified by other more reliable means, Text message, in person etc.

Of course these are just my thoughts on the matter. Other opinions are welcomed.

**Comment by RobH779** [comment]
Yeah that's fair enough. 
I agree with your thoughts on notifications, I would almost certainly turn them off myself. 

Just thought it would be better for workflow to send out an email of the job (maybe via a checkbox "Send Email Notification" in Assign To menu) for less regular users to prevent having to enter it in and then contact them anyway separately to notify them.

Not critical of course and if it requires a paid service then probably not worth it!

**Comment by Charlie9830** [comment]
I agree that would be helpful. I'll look into after the dust settles on Round 1 Release.
